### PR TITLE
Overflow and Text Editing fix

### PR DESCRIPTION
**Pull request by kokodavid**

### CODE REVIEW

File: app/lib/features/onboarding/pages/login_page.dart [hunk 1]
```diff
@@ -133,16 +133,12 @@ class _LoginPageState extends ConsumerState<LoginPage> {
...
                             }
                             return null;
                           },
-                          onChanged: (value) {
-                            username.text = value;
-                            username.selection = TextSelection.fromPosition(
-                              TextPosition(offset: username.text.length),
-                            );
-                          },
+                          onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/login_page.dart:141]: do we need these at all then? [isn't this optional](https://api.flutter.dev/flutter/material/TextFormField/onChanged.html) ?
```suggestion
```

File: app/lib/features/onboarding/pages/login_page.dart [hunk 2]
```diff
@@ -172,16 +168,12 @@ class _LoginPageState extends ConsumerState<LoginPage> {
...
                             }
                             return null;
                           },
-                          onChanged: (value) {
-                            password.text = value;
-                            password.selection = TextSelection.fromPosition(
-                              TextPosition(offset: password.text.length),
-                            );
-                          },
+                          onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/login_page.dart:176]:
```suggestion
```

File: app/lib/features/onboarding/pages/register_page.dart [hunk 3]
```diff
@@ -167,12 +167,7 @@ class _RegisterPageState extends ConsumerState<RegisterPage> {
                                 }
                                 return null;
                               },
-                              onChanged: (value) {
-                                name.text = value;
-                                name.selection = TextSelection.fromPosition(
-                                  TextPosition(offset: name.text.length),
-                                );
-                              },
+                              onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/register_page.dart:170]:
```suggestion
```

File: app/lib/features/onboarding/pages/register_page.dart [hunk 4]
```diff
@@ -222,12 +217,7 @@ class _RegisterPageState extends ConsumerState<RegisterPage> {
                                 }
                                 return null;
                               },
-                              onChanged: (value) {
-                                username.text = value;
-                                username.selection = TextSelection.fromPosition(
-                                  TextPosition(offset: username.text.length),
-                                );
-                              },
+                              onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/register_page.dart:220]:
```suggestion
```

File: app/lib/features/onboarding/pages/register_page.dart [hunk 5]
```diff
@@ -267,12 +257,7 @@ class _RegisterPageState extends ConsumerState<RegisterPage> {
                                 }
                                 return null;
                               },
-                              onChanged: (value) {
-                                password.text = value;
-                                password.selection = TextSelection.fromPosition(
-                                  TextPosition(offset: password.text.length),
-                                );
-                              },
+                              onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/register_page.dart:260]:
```suggestion
```

File: app/lib/features/onboarding/pages/register_page.dart [hunk 6]
```diff
@@ -310,12 +295,7 @@ class _RegisterPageState extends ConsumerState<RegisterPage> {
                                 }
                                 return null;
                               },
-                              onChanged: (value) {
-                                token.text = value;
-                                token.selection = TextSelection.fromPosition(
-                                  TextPosition(offset: token.text.length),
-                                );
-                              },
+                              onChanged: (value) {},
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/register_page.dart:298]:
```suggestion
```

File: app/lib/features/onboarding/pages/start_page.dart [hunk 7]
```diff
@@ -299,15 +298,17 @@ class _StartPageState extends State<StartPage> {
                             onBoardingPages[index].image,
                           ),
                         ),
+                        const SizedBox(
```
Comment (gnunicorn) [inline app/lib/features/onboarding/pages/start_page.dart:301]: I have no idea if that fixed it. Can you provide a screenshot?

File: app/pubspec.lock
Comment (gnunicorn) [inline app/pubspec.lock:1]: these seem unrelated...

**Review by gnunicorn — APPROVED** [review]
Seems fine, but screenshots would be good to confirm it works now as expected.

also: want to run my integration tests against this first, as well ... that covers exactly these two screens...

File: app/lib/features/onboarding/pages/start_page.dart [hunk 8]
```diff
@@ -299,15 +298,17 @@ class _StartPageState extends State<StartPage> {
...
+                          height: 15,
```
Comment (gtalha07) [inline app/lib/features/onboarding/pages/start_page.dart:302]: Screenshots will be much better to get overview of the fix, I'm doubtful if hardcoding constraints will prevent overflow as it might appear again on different sized device, try using ```Flex()``` maybe and keep ```mainAxisSize.min```?

**Comment by gnunicorn** [comment]
tests pass. but at least the text is still unlegibable for me:

grafik

File: app/lib/features/onboarding/pages/start_page.dart (hunk 8 above)
Comment (kumarpalsinh25) [reply app/lib/features/onboarding/pages/start_page.dart:302]: I would suggest here to use `Expanded` widget with appropriate `flex` value instead of `SizeBox` with `hardcoded values`.

File: app/lib/features/onboarding/pages/login_page.dart (hunk 1 above)
Comment (kumarpalsinh25) [reply app/lib/features/onboarding/pages/login_page.dart:141]: Yes,` onChanged: (value) {},` is optional and can be remove.

**Review by kumarpalsinh25 — APPROVED** [review]
Looks good overall.  
Just need to manage some hardcoded size values as suggested.

File: app/lib/features/onboarding/pages/start_page.dart (hunk 7 above)
Comment (kokodavid) [reply app/lib/features/onboarding/pages/start_page.dart:301]: I included the screenshots in the respective issues but I I will included them here too .

File: app/lib/features/onboarding/pages/start_page.dart (hunk 8 above)
Comment (kokodavid) [reply app/lib/features/onboarding/pages/start_page.dart:302]: Well Noted ,  I'll go ahead and make the changes.

Comment (kokodavid) [reply app/lib/features/onboarding/pages/start_page.dart]: Screenshots for the fix .
image
overflow fix
